solcore/absorption_calculator/tmm_core_vec.py

```diff
@@ -630,13 +639,13 @@ def inc_group_layers(n_list, d_list, c_list):
     * num_layers = number of layers total
     """
 
-    if (n_list.ndim != 1) or (d_list.ndim != 1):
+    if (d_list.ndim != 1):
         raise ValueError("Problem with n_list or d_list!")
```

Comment:
Given the change in the condition, this exception should be just "Problem with d_list!")